server/superdesk/io/commands/update_ingest_tests.py

```diff
@@ -225,6 +225,26 @@ def test_query_getting_expired_content(self):
             expiredItems = get_expired_items(provider, now)
             self.assertEquals(4, expiredItems.count())
 
+            # actualy ingest the items and expire them
+            self.ingest_items(items, provider)
+            before = service.get(req=None, lookup={})
+            self.assertEqual(6, before.count())
+
+            # four files in grid fs
+            current_files = app.media.fs('upload').find()
+            self.assertEqual(4, current_files.count())
+
+            remove = RemoveExpiredContent()
+            remove.run(provider.get('type'))
+
+            # only one left in ingest
+            after = service.get(req=None, lookup={})
+            self.assertEqual(1, after.count())
+
+            # all gone
+            current_files = app.media.fs('upload').find()
+            self.assertEqual(0, current_files.count())
```

Comment:
In general each unit test case should only test a single use case (i.e. one execution path), resulting in more fine-grained failure messages, pinpointing the exact execution path that doesn't work. I would thus suggest splitting this test into four separate test cases.

In general it's good though.